microshift_troubleshooting/microshift-troubleshoot-updates.adoc

```diff
@@ -8,20 +8,15 @@ toc::[]
 
 To troubleshoot {microshift-short} updates, use the following guide.
 
-//[IMPORTANT]
-//====
-//You can only update {microshift-short} from one minor version to the next in sequence. For example, you must update 4.14 to 4.15.
-//====
-
 include::modules/microshift-updates-troubleshooting.adoc[leveloffset=+1]
 
 //additional resources for troubleshooting module
 [role="_additional-resources"]
 .Additional resources
 * xref:../microshift_install_get_ready/microshift-greenboot.adoc#microshift-greenboot-systemd-journal-data_microshift-greenboot[Enabling `systemd` journal service data persistency]
 * xref:../microshift_troubleshooting/microshift-version.adoc#microshift-version[Checking the MicroShift version]
-* xref:../microshift_install/microshift-install-rpm.adoc#microshift-service-stopping_microshift-install-rpm[Stopping the MicroShift service]
-* xref:../microshift_install/microshift-install-rpm.adoc#microshift-service-starting_microshift-install-rpm[Starting the MicroShift service]
+* xref:../microshift_install_rpm/microshift-install-rpm.adoc#microshift-service-stopping_microshift-install-rpm[Stopping the MicroShift service]
+* xref:../microshift_install_rpm/microshift-install-rpm.adoc#microshift-service-starting_microshift-install-rpm[Starting the MicroShift service]
```

Comment:
This xref takes you to "Installing from an RPM package" instead of directly to "Starting the MicroShift service".